macos/FluentUI/Link/Link.swift

```diff
@@ -126,6 +126,15 @@ open class Link: NSButton {
 		}
 	}
 
+	private let cornerRadius: CGFloat = 2
```

Comment:
nit: Not related to your specific change, but since you're changing the file, could you move the private members to the bottom following the [swift-guide (FileContentOrdering) guidelines](https://github.com/microsoft/swift-guide/blob/gh-pages/FileContentOrdering.md)?